_episodes/23-continuous-integration-automated-testing.md

```diff
@@ -71,8 +71,8 @@ so it's worth taking a bit of time looking into this file format.
 [YAML](https://www.commonwl.org/user_guide/yaml/)
 (a recursive acronym which stands for "YAML Ain't Markup Language")
 is a language designed to be human readable.
-The three basic things you need to know about YAML to get started with GitHub Actions are
-key-value pairs, arrays, and maps.
+The four basic things you need to know about YAML to get started with GitHub Actions are
```

Comment:
```suggestion
A few basic things you need to know about YAML to get started with GitHub Actions are
```

Reply:
Perhaps we should not count the things as clearly they change :-).

Reply:
Yes, I think this is a good amendment!